include/LightGBM/utils/common.h

```diff
@@ -588,6 +588,15 @@ inline void check_elements_interval_closed(const float *y, float ymin, float yma
   }
 }
 
+// Check that all y[] are in interval [ymin, ymax] (end points included); throws error if not
+inline void check_int32_elements_interval_closed(const int32_t *y, int32_t ymin, int32_t ymax, int32_t ny, const char *callername) {
```

Comment:
Our function naming is the "Camel Case" style.

Reply:
I made it resemble the existing function in "Snake Case" style:
```c++
// Check that all y[] are in interval [ymin, ymax] (end points included); throws error if not
inline void check_elements_interval_closed(const float *y, float ymin, float ymax, int ny, const char *callername) {
  for (int i = 0; i < ny; ++i) {
    if (y[i] < ymin || y[i] > ymax) {
      Log::Fatal("[%s]: does not tolerate element [#%i = %f] outside [%f, %f]", callername, i, y[i], ymin, ymax);
    }
  }
}
```

I found the other function in "Snake Case" style:
```c++
// One-pass scan over array w with nw elements: find min, max and sum of elements;
// this is useful for checking weight requirements.
inline void obtain_min_max_sum(const float *w, int nw, float *mi, float *ma, double *su)
```

Reply:
I think the two functions should resemble the other functions defined in` common.h` in style as the following:
```c++
template <typename T>
inline void CheckElementsIntervalClosed(const T *y, T ymin, T ymax, int ny, const char *callername)
```

```c++
template <typename T>
inline void ObtainMinMaxSum(const T *w, int nw, T *mi, T *ma, double *su)
```

Reply:
I think you should use [cpplint](https://github.com/google/styleguide/tree/gh-pages/cpplint).

Reply:
yeah, you can edit these two function. for the cpplint, we can use it in the future.